packages/fast-colors/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@microsoft/fast-colors",
+  "version": "1.0.0",
+  "description": "A color utility for generating a range of colors from a single color",
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "contributors": [
+    {
+      "name": "Fluent Web",
+      "email": "fwq@microsoft.com"
+    },
+    {
```

Comment:
Are we doing this? May be difficult to maintain as more contributors jump in. I thought we were going to do inline and in readme? Just curious.

Reply:
Yea not sure - I thought I'd call them out directly as this is essentially a port of their codebase but we can discuss. Thoughts Aaron?

Reply:
@awentzel ^ thoughts?

Reply:
We were going add a section in the readme to call them out specifically.  The package.json file then doesn't have to change. Additionally, I will add them as contributors to the project.

Reply:
You probably will want to ask them what email/profile to use for their work.

Reply:
Okay sounds good - I'll add it to the Readme. I discussed with bobby point to their GitHub handles, so I'll reference their profile pages

Reply:
@chrisdholt @awentzel Updates made.